.gitignore

```diff
@@ -112,3 +112,8 @@ ENV/
 
 
 .DS_Store
+
+#Config file and generated openapi doc
```

Comment:
Given the name can be user-generated I think we don't need these specific exclusions

Reply:
Yeah, will remove them.

Reply:
removed